src/clj/game/cards/agendas.clj

```diff
@@ -165,6 +165,13 @@
     :end-turn {:effect (effect (lose :runner :tag 2))
                :msg "make the Runner lose 2 tags"}}
 
+   "CFC Mining Contract"
```

Comment:
This card's English title is "CFC Excavation Contract"  ([link]()http://www.cardgamedb.com/index.php/netrunner/android-netrunner-card-spoilers/_/red-sand-cycle/crimson-dust/))